### PR TITLE
[1.x] Create ci.yml

### PR DESCRIPTION
**Pull request by mgamal92**
| Q             | A                                                                  |
|---------------|--------------------------------------------------------------------|
| Branch?       | main                                              |
| Bug fix?      | yes                                                             |
| New feature?  | yes                   |
| Deprecations? | no  |
| Tickets       | Fix #...            |
| License       | MIT                                                                |

### CODE REVIEW

**Comment by imdhemy** [comment]
Let's wait on this until we replace PHPunit with Pest.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: PHP Composer
+
+on:
+  push:
```
Comment (imdhemy) [inline .github/workflows/ci.yml:4]: Let's run this workflow on all branches.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    - name: Install dependencies
+      run: composer install --prefer-dist --no-progress
+
+    - name: "Coding Standard"
+      run: "composer psalm"
+      
+    - name: "Code Format"
+      run: "composer format"
+#
```
Comment (imdhemy) [inline .github/workflows/ci.yml:32]: We can uncomment this step now.

**Review by imdhemy — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (mgamal92) [reply .github/workflows/ci.yml:4]: Done

File: .github/workflows/ci.yml (hunk 2 above)
Comment (mgamal92) [reply .github/workflows/ci.yml:32]: Why ?

Comment (imdhemy) [reply .github/workflows/ci.yml:32]: To enable tests 😅

**Review by imdhemy — APPROVED** [review]
Thank you 🙌

**Pull request merged**